docs/content/en/docs/reference/metrics.md

```diff
@@ -0,0 +1,422 @@
+---
+title: "Metrics Reference"
+description: >
+  This reference documents Prometheus metrics exposed by Tetragon.
+weight: 4
+---
+<!-- This file is autogenerated via https://github.com/isovalent/metricstool -->
+
+# Tetragon Health Metrics
```

Comment:
Could you make it a level 2 title (with all the other level 1 titles), otherwise it breaks the hierarchy and the formatting of the page 
```suggestion
## Tetragon Health Metrics
```

For example later:
<details><summary>Click to see preview</summary>
<p>
<img width="742" alt="image" src="https://github.com/cilium/tetragon/assets/11256051/f774882b-7e6e-43b6-bc17-559cd4833a0c">
</p>
</details> 


It does not expect a level 1 after the page level 1 that is already the title from the frontmatter.